src/docs.md

```diff
@@ -3,87 +3,90 @@ title: Flutter Documentation
 description: The landing page for Flutter documentation.
 ---
 
+<div class="card-deck">
 {% for card in site.data.docs_cards %}
-- ### [{{card.name}}]({{card.url}})
-  {:.card-title.no_toc}
-
-  {{card.description}}
-{:.card}
+    <a class="card" href="{{card.url}}">
+        <div class="card-body">
+            <h5 class="card-title">{{card.name}}</h5>
```

Comment:
How about using `<header>` here instead of a (somewhat arbitrary) `<h5>` (or `<h4>` as is used on site-www.